### PR TITLE
images: release 0.20.0

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

**Review by bart0sh — APPROVED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -58,9 +58,9 @@ endif
 
 test-with-kind:
 	@build/docker/build-image.sh intel/intel-fpga-admissionwebhook buildah
-	@$(PODMAN) tag localhost/intel/intel-fpga-admissionwebhook:devel docker.io/intel/intel-fpga-admissionwebhook:devel
+	@$(PODMAN) tag localhost/intel/intel-fpga-admissionwebhook:0.20.0 docker.io/intel/intel-fpga-admissionwebhook:0.20.0
```
Comment (rojkov) [inline Makefile:61]: Perhaps `0.20.0` could be replaced with $(TAG) here?

Comment (rojkov) [reply Makefile:61]: Oops, NVM. This should be fixed in the master branch first.

Comment (mythi) [reply Makefile:61]: @rojkov speaking of `test-with-kind`: I noticed there's a kind Github action. might be interesting to enable this for PRs.

**Comment by mythi** [comment]
@rojkov @bart0sh this is OK to merge

**Pull request merged**